src/lgr_advanced/lgr_editor/forms/importer.py

```diff
@@ -5,6 +5,7 @@
 from django.core import validators
 from django.utils.translation import ugettext_lazy as _
 
+from lgr_models.models import RzLgr
 from .fields import (DEFAULT_UNICODE_VERSION,
                      VALIDATING_REPERTOIRES,
```

Comment:
is this still used?

Reply:
Yes in the CreateLGRForm (/a/editor/new/). It was specify in the ticket to only change /a/editor/import/. Don't know if it makes sense, though.